ee/localserver/dt4a.go

```diff
@@ -48,17 +50,34 @@ func (ls *localServer) requestDt4aInfoHandlerFunc(w http.ResponseWriter, r *http
 		ls.accelerate(r.Context())
 	}
 
-	dt4aInfo, err := ls.knapsack.Dt4aInfoStore().Get(localserverDt4aInfoKey)
+	if r.Header.Get(dt4aAccountUuidHeaderKey) != "" {
+		dt4aInfo, err := ls.knapsack.Dt4aInfoStore().Get([]byte(r.Header.Get(dt4aAccountUuidHeaderKey)))
```

Comment:
If I read this right, it's storing in a hash with `<accountUUID>` as the key. How does this work with the possible addition of `<userUUID>`

Reply:
This PR makes no provisions for user uuid other than setting it the request header. My understanding is that user uuid is purely informational from launcher perspective at this point. I plan to determine where to discover that information on the k2 side and log mis-matches in a follow on PR.

Reply:
I'm thinking about what kind of structure this assumes k2 will emit today, and in the future. And I feel pretty unsure

Reply:
Eh. I'm letting perfect distracted from good enough. James says this is how k2 works, so let's leave it